lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -1688,104 +1632,6 @@ GLStatus TFGraphFunctionLowering::createDatasetCreationContext(
   return GLStatus::Success;
 }
 
-// TODO: Remove this version when graph op takes over completely.
-template <>
-GLStatus TFGraphFunctionLowering::createDatasetCreationContext(
```

Comment:
i'm fine with removing this legacy code path. Please document it in PR description.

Reply:
Updated PR description.